client/src/components/Utils/Toast.tsx

```diff
@@ -0,0 +1,28 @@
+import { Toast as FlowbiteToast } from 'flowbite-react';
+import { HiCheck } from 'react-icons/hi';
+import { toast } from '../../redux/feats/globalSlice/globalSlice';
+import { useAppDispatch } from '../../redux/store';
+interface ToastProps {
+    message: string;
+}
+
+const Toast: React.FC<ToastProps> = ({ message }) => {
```

Comment:
`nit:` It looks like using `React.FC` is discouraged. See this [link](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/).